tldw_Server_API/app/core/Audit/unified_audit_service.py

```
@@ -3468,67 +3508,74 @@ def _rows_to_csv(rows: list[dict[str, Any]]) -> str:
         p.write_text(_rows_to_csv(all_rows), encoding="utf-8")
         return len(all_rows)
 
-    def _determine_category(self, event_type: AuditEventType) -> AuditEventCategory:
-        """Auto-determine category from event type"""
-        type_name = event_type.name.lower()
+    def _determine_category(self, event_type: AuditEventType | str) -> AuditEventCategory:
```

Ruff (0.15.9)

[warning] Too many return statements (16 > 6)

(PLR0911)